go/cmd/kube-vpnkit-forwarder/main.go

```diff
@@ -1,28 +1,42 @@
 package main
 
 import (
+	"context"
 	"flag"
-	"log"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
 
 	"github.com/moby/vpnkit/go/pkg/controller"
 	"github.com/moby/vpnkit/go/pkg/vpnkit"
+	log "github.com/sirupsen/logrus"
```

Comment:
`logrus` was being used in the controller but not here - switched for consistency and to add ability to config log level via flag